src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -99,11 +113,11 @@ def pull_helm_chart(registry_path, kube_config, kube_context, helm_client_locati
     _, error_helm_chart_pull = response_helm_chart_pull.communicate()
     if response_helm_chart_pull.returncode != 0:
         telemetry.set_exception(exception=error_helm_chart_pull.decode("ascii"), fault_type=consts.Pull_HelmChart_Fault_Type,
-                                summary='Unable to pull helm chart from the registry')
-        raise CLIInternalError("Unable to pull helm chart from the registry '{}': ".format(registry_path) + error_helm_chart_pull.decode("ascii"))
+                                summary="Unable to pull {} helm charts from the registry".format(chart_name))
+        raise CLIInternalError("Unable to pull {} helm chart from the registry '{}': ".format(chart_name, registry_path) + error_helm_chart_pull.decode("ascii"))
 
 
-def export_helm_chart(registry_path, chart_export_path, kube_config, kube_context, helm_client_location):
+def export_helm_chart(registry_path, chart_export_path, kube_config, kube_context, helm_client_location, chart_name='azure-arc'):
```

Comment:
should the chart_name be azure-arc-k8sagents instead

Reply:
In the previous code , we were using azure-arc only , so kept it the same way

Reply:
Okay